primitives/src/identity_claim.rs

```diff
@@ -1,38 +1,81 @@
 use crate::{identity_id::IdentityId, Moment};
 use codec::{Decode, Encode};
-use sp_std::prelude::Vec;
+use sp_std::prelude::*;
 
 /// All possible claims in polymesh
 #[derive(Encode, Decode, Clone, PartialEq, Eq, Debug, PartialOrd, Ord)]
-pub enum IdentityClaimData {
+pub enum Claim {
     /// User is Accredited
-    Accredited(IdentityId),
+    Accredited,
     /// User is Accredited
-    Affiliate(IdentityId),
+    Affiliate,
     /// User has an active BuyLockup (end date defined in claim expiry)
-    BuyLockup(IdentityId),
```

Comment:
Why was the target did removed from all claims?

Reply:
A merge mistake! Thanks @maxsam4 .

Reply:
Because it was duplicated info:

    Target identifier is already stored as part of the first key in Identity::Claims (double_map).
    Issuer identifier is already stored as part of second key in Identity::Claims (double_map).

Using that new Identity::Claims definition, we improve the fetching of claims.

Reply:
I think there's a confusion here, we need three dids and calling all of them target is turning out to be a bad idea xD. The 3 dids required represent:
1) Claim Receiver (part of the first key in Identity::Claims)
2) Claim Issuer (part of the second key in Identity::Claims)
3) Target/Context/Relevant DID. This will be either the did of a token or of a token issuer. This was stored in claims in form of `BuyLockup(IdentityId)`.

The idea is that "Alice" will be able to issuer a claim on "Bob" saying "Bob" is Whitelisted in "Charlier's token".
"Alice" is claim issuer
"Bob" is claim receiver
"Charlier's token" is Target/Context/Relevant DID.

Reply:
@maxsam4 I used the following semantic for claims:
1.  Claim Receiver -> **target**
2.  Claim Issuer -> **issuer**
3.  Target/Context/Relevant DID -> **scope**